crypto/crypto_test.cc

```diff
@@ -73,6 +73,25 @@ TEST(CryptoTest, Strndup) {
   EXPECT_STREQ("", str.get());
 }
 
+TEST(CryptoTest, OPENSSL_hexstr2buf) {
+  const char *test_cases[][2] = {{"a2", "\xa2"},
```

Comment:
nit: we could get some additional coverage cheaply here

- null inputs
- odd-length `str`
- non-hex digits